example/react-native/README.md

```diff
@@ -8,17 +8,24 @@ This is a small example app to demonstrate the features of the AeroGear SDK for
 Make sure that npm and the react native CLI are installed on your system:
 
 ```
-$ brew install npm
-$ npm install -g react-native
+brew install npm
+npm install -g react-native-cli
 ```
 
 ## Try it out
 
+Make sure you have the 'core' package built first. See [../../docs/contributing-guide.adoc](../../docs/contributing-guide.adoc)
+
 You can try the example by running:
 
 ```
-$ npm install
-$ react-native run-<platform>
+# Install dependencies
+# NOTE: This will also install a copy of the core sdk from ../../packages/core
```

Comment:
I will try to automate that so all packages will be automatically linked by example.
Have couple ideas how we can use lerna for this particular use case.